### ISSUE TITLE
Webapp should notify the user when the poller is down

### ISSUE THREAD

**Issue by kaiwren** [srushti/goldberg#88]
Ideally, like CC.rb, the 'force build' button should be replaced with a 'start poller' button.

**Comment by srushti** [comment]
I'm thinking we'll have the poller posting a timestamp to one row in the db or touching a file every 30 seconds or so, and the web app can see if that timestamp is greater than 35 seconds or something. What do you think?
Also, I'd like to have a separate issue to deal with starting the poller from the web app directly.

**Comment by kaiwren** [comment]
Agreed on point one - I'd favour the touch/pidfile approach. Check if a process with that ID is actually running and if not delete the pid file and mark poller as down. We'll need CLI commands for all this (basically what we have as an alias on the server now).

Also agreed on point two - moved out into #102.
